src/stores/systemPreferences.ts

```diff
@@ -95,6 +100,10 @@ function capitalize(str: string) {
   return str.charAt(0).toUpperCase() + str.slice(1)
 }
 
-export function buildSystemPreferencesItems() {
-  return preferences.map(buildSystemPreferenceItem)
+export function buildSystemPreferencesItems(
```

Comment:
It currently only gets called once. But only after the Apps have been fetched, because I need to find the SystemPreferences url to be able to get the correct icon.